seeds.rb

```diff
@@ -24,12 +24,12 @@
   artist: "Silento"},
   {
   title: "Can't Feel My Face",
-  artist: "The Weeknd"},
+  artist: "The Weekend"},
   {
   title: "Locked Away",
-  artist: "R. City Featuring Adam Levine"}
+  artist: "R. City Featuring Adam Levine"},
   ]
 
 songs.each do |song|
-  Song.create! title: song[:title], artist: song[:artist], suggested_by: User.all.sample 
+  Song.create! title: song[:title], artist: song[:artist], suggested_by: User.find_by(email: "devin@example.com")
```

Comment:
I think we probably want to leave this random. You can always change them in your own console if you want.